src/service/layers-impl.js

```diff
@@ -658,12 +755,66 @@ export class LayoutElement {
    * @return {!PositionDef}
    */
   getOffsetFromParent() {
-    if (this.needsRemeasure_) {
-      this.remeasure();
-    }
+    this.remeasure();
     return this.position_;
   }
 
+  /**
+   * Whether the layout is a descendant of the most recently scrolled layer.
+   *
+   * This MUST NOT be called unless inside an #ancestry iterator function.
+   * "Activeness" is only determined for the lifespan of the #ancestry call.
+   *
+   * @return {boolean}
+   */
+  isActive() {
```

Comment:
`isActiveUnsafe`?

Reply:
Because you shouldn't call it unless you really know what you're doing. Similar to "unsafe" suffixes in resources-impl.js.